srcpkgs/asahi-base/files/README.voidlinux

```diff
@@ -0,0 +1,24 @@
+For some time, the kernel, mesa and uboot packages will have Asahi
+as upstream.  The Asahi Linux documentation is an excellent source.
+Here we discuss Void Linux specific aspects only.
+
+Use Asahi Linux install scripts with "UEFI environment only", then proceed
+with usual Void installation.  The default is to have EFI partition at
+/boot, other choices can be made by editing m1n1 config.
+
+At the very least, install base-system, asahi-base and an initram
+generator.  By default, also asahi-uboot, grub-arm64-efi and asahi-scripts
+are required, unless one changes initram or payload for m1n1.
+
+We ship a kernel hook for m1n1, modeled upon Asahi upstream.  This allows
+to either just use m1n1, or use it in combination with uboot and
+grub (default) by setting the PAYLOAD variable (requires asahi-uboot,
+grub-arm64-efi).  We ship tinyramfs config and hooks (to use, add asahi
+to hooks).  The default is dracut, its hooks are provided by asahi-scripts
+together with mkinitcpio ones.
+
+If using audio, must install asahi-audio, enable speakersafetyd service,
```

Comment:
speakersafetyd is enabled by default by the INSTALL script, maybe "ensure the speakersafetyd service is enabled"

Reply:
done